src/redux/constants/registration.constant.js

```diff
@@ -0,0 +1,6 @@
+export const REGISTER_REQUESTED = "REGISTER_REQUESTED";
```

Comment:
Put this under action types, we agreed to use the same file.

Reply:
done